formats/html_publication/publisher/details.json

```diff
@@ -21,6 +21,12 @@
     },
     "first_published_version": {
       "type": "boolean"
+    },
+    "isbn": {
+      "type": "string"
+    },
+    "print_meta_data_contact_address": {
```

Comment:
The descriptions you added to the PR should be included as descriptions in the schema